portlets/src/main/webapp/vue-app/realizations/components/RealizationItem.vue

```diff
@@ -30,7 +30,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       </span> 
     </td>
     <td class="text-truncate align-center">
-      {{ domainDescription }}
+      <span v-sanitized-html="domainDescription"></span>
     </td>
```

Comment:
@GouadriaHanen you can instead of addin a new DOM element ( when cumulating all of them, it reduces a lot the page loading performances ), you can add `v-sanitized-html` for `td`.
```html
    <td class="text-truncate align-center" v-sanitized-html="domainDescription">
    </td>

```
@AzmiTouil FYI